demo/helpers/cosmoz-demo-view.js

```diff
@@ -17,9 +17,9 @@ class CosmozDemoView extends PolymerElement {
 		</style>
 		<div class="flex text">{{ item.id }}</div>
 		<div>
-			<paper-icon-button slot="actions" disabled$="[[ prevDisabled ]]" icon="chevron-left" cosmoz-data-nav-select="-1"></paper-icon-button>
+			<paper-icon-button slot="actions" disabled$="[[ prevHidden ]]" icon="chevron-left" cosmoz-data-nav-select="-1"></paper-icon-button>
```

Comment:
Should this be:
```suggestion
			<paper-icon-button slot="actions" hidden$="[[ prevHidden ]]" icon="chevron-left" cosmoz-data-nav-select="-1"></paper-icon-button>
```
?

Reply:
Yes, edited.